src/ragapp/backend/controllers/system_prompt.py

```diff
@@ -1,32 +1,14 @@
 import os
-from string import Template
 
 import dotenv
 
 from backend.constants import (
     DEFAULT_SYSTEM_PROMPT,
     ENV_FILE_PATH,
-    SYSTEM_PROMPT_WITH_TOOLS_TPL,
 )
 
 
 class SystemPromptManager:
```

Comment:
what is the role of the systempromptmanager now that we moved the system prompt out of the chat config? can't AgentPromptManager take it's responsibility?

Reply:
Removed SystemPromptManager. the system prompt are now from Agent config